Classes/YTPlayerView.m

```diff
@@ -691,17 +699,20 @@ - (BOOL)loadWithPlayerParams:(NSDictionary *)additionalPlayerParams {
   if ([playerParams objectForKey:@"playerVars"]) {
     NSMutableDictionary *playerVars = [[NSMutableDictionary alloc] init];
     [playerVars addEntriesFromDictionary:[playerParams objectForKey:@"playerVars"]];
-      
-    if (![playerVars objectForKey:@"origin"]) {
-        self.originURL = [NSURL URLWithString:@"about:blank"];
-    } else {
-        self.originURL = [NSURL URLWithString: [playerVars objectForKey:@"origin"]];
-    }
   } else {
-    // This must not be empty so we can render a '{}' in the output JSON
     [playerParams setValue:[[NSDictionary alloc] init] forKey:@"playerVars"];
   }
```

Comment:
These seems to be no-op with the newly added lines below.

Reply:
yea, thanks for noticing.